src/kernel/mempool_entry.h

```diff
@@ -170,6 +170,15 @@ class CTxMemPoolEntry
     const Parents& GetMemPoolParentsConst() const { return m_parents; }
```

Comment:
In commit 79bcc5ca0679daf6e57fc4d7ce2244262a7cfd13 the last paragraph of its commit message seems like it misses part of a sentence.

Reply:
Updated the commit message